coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/metadata/sql/PostgresDatasetMap.scala

```diff
@@ -806,10 +806,10 @@ trait BasePostgresDatasetMapWriter[CT] extends BasePostgresDatasetMapReader[CT]
     using(conn.prepareStatement(unsafeReloadDatasetQuery)) { stmt =>
       stmt.setLong(1, newDatasetInfo.nextCounterValue)
       stmt.setLong(2, newDatasetInfo.latestDataVersion)
-      stmt.setDatasetId(3, newDatasetInfo.systemId)
-      stmt.setString(4, newDatasetInfo.localeName)
-      stmt.setBytes(5, newDatasetInfo.obfuscationKey)
-      stmt.setString(6, newDatasetInfo.resourceName.orNull) // resource_name is a nullable column
+      stmt.setString(3, newDatasetInfo.localeName)
+      stmt.setBytes(4, newDatasetInfo.obfuscationKey)
+      stmt.setString(5, newDatasetInfo.resourceName.orNull) // resource_name is a nullable column
+      stmt.setDatasetId(6, newDatasetInfo.systemId)
```

Comment:
I don't understand the purpose of this change.

Reply:
This function was previously unused and was broken.

Reply:
🤦